resources/lang/fr/manager/notification.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+return [
+    /*
+    * --------------------------------------------------------------------------
+    * Manager Notification Localizations
+    * --------------------------------------------------------------------------
+    * Route: /manager/profile
+    *        /manager/faq
+    *        /manager/jobs
+    * ViewComposer: DemoNotificationViewComposer.php
+    * View: views/manager/notification.html.twig
+    */
+    'first' => 'Ceci est un compte démo.',
+    'link' => [
+        'text' => 'Cliquez ici',
+        'title' => "Cliquez ici pour savoir comment publier une offre d'emploi sur Talent Cloud",
+        'href' => '#miseàjour'
```

Comment:
Unless we're also localizing id's in the twig template, this isn't going to work